Source/SPUD/Private/SpudSubsystem.cpp

```diff
@@ -211,6 +213,14 @@ void USpudSubsystem::OnPostLoadMap(UWorld* World)
 			       TEXT("OnPostLoadMap restore: %s"),
 			       *LevelName);
 
+			// Switch to loading state if not already. This is needed because events triggered by loading, perhaps at some remote (event 1 > event 2 > etc)
+			// might need to check if SPUD is currently loading the game.
+			if (CurrentState != ESpudSystemState::LoadingGame)
+			{
+				CurrentState = ESpudSystemState::LoadingGame;
+				PreLoadGame.Broadcast(FString());
+			}
+
```

Comment:
This change is going to do the incorrect thing when you load a new map just in the course of normal play. OnPostLoadMap will be called in this case and this change will make SPUD think it's loading a save game when it's not.